legal-api/src/legal_api/core/filing.py

```diff
@@ -464,7 +464,8 @@ def _add_ledger_order(filing: FilingStorage, ledger_filing: dict) -> dict:
             ledger_filing["data"] = {}
         ledger_filing["data"]["order"] = court_order_data
 
-    def _is_display_ledger(self, filing: FilingStorage) -> bool:
+    @staticmethod
+    def _is_display_ledger(filing: FilingStorage) -> bool:
         """Return boolean that display the ledger."""
         return filing.filing_type != Filing.FilingTypes.ADMIN_FREEZE
```

Comment:
This doesn't appear to be used as a class method, and so it was not recognizing that `filing` was passed correctly on line 443. I have remove `self` and marked it as a static method